tests/test_edi_835_files/sample_835.txt

```diff
@@ -152,14 +153,15 @@ REF*28*104938~
 REF*SY*299325566~
 DTM*050*20211231~
 AMT*AU*157~
+AMT*I*5.1~
```

Comment:
is there a test case we could do where there is no interest amount? (ex: not having an `AMT*I`)